src/components/ec-timer/ec-timer.story.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
```

Comment:
We should definitely use ArgsTable and CustomSource in these stories. Otherwise the mdx format has less features than writing stories in plain JS using just args.

What is actually a benefit of using mdx in comparison to plain js?

Reply:
I added some ArgsTable to the example.

But tbh I don't understand the CustomSource.

The people from UX will be ones to write this doc pages I think will be easier for them to write it on mdx.